src/Runner.Worker/ContainerOperationProvider.cs

```diff
@@ -494,31 +493,14 @@ private void ContainerRegistryLogout(string configLocation)
 
         private void UpdateRegistryAuthForGitHubToken(IExecutionContext executionContext, ContainerInfo container)
         {
-            var registryIsTokenCompatible = container.RegistryServer.Equals("docker.pkg.github.com", StringComparison.OrdinalIgnoreCase);
+            var registryIsTokenCompatible = container.RegistryServer.Equals("ghcr.io", StringComparison.OrdinalIgnoreCase);
```

Comment:
any name other than `ghcr.io` we should also check?

Reply:
I don't think so. I'm asking the packages team to take a look first to this PR just in case.

Reply:
If we could check against `containers.pkg.github.com` as well that'd be good. It was the development hostname used before we had `ghcr.io` and shouldn't be used anywhere but internally, but they are aliased to one another so would be good to have that corner case covered.

Reply:
We also should leave `docker.pkg.github.com` as an option, since it hasn't been sunset yet and I believe this will break some current workflows that use GITHUB_TOKEN with the legacy registry

Reply:
@gimenete do you mind add those 2 strings to the checklist? i think we are good to merge after this.

Reply:
👍 I've just pushed a change to do the same if the server is `containers.pkg.github.com`